definitions/features/foreman_server.rb

```diff
@@ -9,9 +9,11 @@ class ForemanServer < ForemanMaintain::Feature
       end
 
       def services
-        [
-          system_service('httpd', 30)
-        ]
+        if foreman_service_installed?
```

Comment:
If `foreman-service` is installed, Apache is usually still used as a reverse proxy. The installer can set it up without Apache, but I personally haven't used that and we don't recommend it unless you know what you're doing.

Reply:
> If `foreman-service` is installed, Apache is usually still used as a reverse proxy.

Yes, Apache will be there. To handle this, I have added Apache feature separately which having `httpd` in services list.

Reply:
here -> https://github.com/theforeman/foreman_maintain/pull/321/files#diff-9c2e92406f1636a749c776ff0cd1b1ceR12